apps/scraper-runner/src/main.ts

```diff
@@ -25,39 +25,49 @@ function createFilename(impacted: string, recommendation: string): string {
 }
 
 function createImpactList(healthRemedies: HealthRemedies[]): ImpactFileList {
-  return healthRemedies.map((healthRemedy) => {
-    return {
-      impacted: healthRemedy.impacted,
-      recommendations: healthRemedy.recommendations.map((recommendation) => {
-        return {
-          fileName: createFilename(healthRemedy.impacted, recommendation),
-          recommendation,
-        };
-      }),
-    };
-  });
+  for (let i = 0; i < healthRemedies.length; i++) {
+    for (
+      let recommendationInd = 0;
+      recommendationInd < healthRemedies[i].recommendations.length;
+      recommendationInd++
+    ) {
+      // console.log(healthRemedies[i].recommendations[recommendationInd]);
+      if (healthRemedies[i].recommendations[recommendationInd]) {
+        const recommendation =
+          healthRemedies[i].recommendations[recommendationInd].recommendation;
+        healthRemedies[i].recommendations[
+          recommendationInd
+        ].filename = createFilename(
+          healthRemedies[i].impacted,
+          recommendation
+        ).replace('/', '_');
```

Comment:
do you also need to replace spaces? e..g if impacted is "brain function" should it be brain_function?